tests/src/test/scala/ch/renku/acceptancetests/workflows/Collaboration.scala

```diff
@@ -18,4 +19,35 @@ trait Collaboration {
     Then("they should see a 'No merge requests to display' info")
     verify userCanSee projectPage.Collaboration.MergeRequests.noMergeRequests
```

Comment:
Not entirely sure if that works properly. If I click the Collaboration tab (line 16) then (at least on the DEV as of 8/6/2020) the Issues tab gets displayed not the Merge Reguests. So shouldn't you first do:
```click on projectPage.Collaboration.MergeRequests.tab```
before you try to verify if the `No merge requests to display.` text is visible?

Reply:
I do not follow this comment. It looks like it does first click on the MergeRequests.tab before checking the text:

```
  def verifyMergeRequestsIsEmpty(implicit projectDetails: ProjectDetails): Unit = {
    val projectPage = ProjectPage()
    When("the user navigates to the Collaboration tab")
    click on projectPage.Collaboration.tab
    And("they navigate to the Merge Requests sub tab")
    click on projectPage.Collaboration.MergeRequests.tab
    Then("they should see a 'No merge requests to display' info")
    verify userCanSee projectPage.Collaboration.MergeRequests.noMergeRequests
  }
```

Am I missing something?

Reply:
You are right. We got tricked by GitHub which skipped the `click on projectPage.Collaboration.MergeRequests.tab` line :)
Sorry